x-pack/plugins/alerting/server/lib/wrap_search_source_client.test.ts

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { loggingSystemMock } from '@kbn/core/server/mocks';
+import { ISearchStartSearchSource } from '@kbn/data-plugin/common';
+import { createSearchSourceMock } from '@kbn/data-plugin/common/search/search_source/mocks';
+import { of, throwError } from 'rxjs';
+import { wrapSearchSourceClient } from './wrap_search_source_client';
+
+const logger = loggingSystemMock.create().get();
+
+const rule = {
+  name: 'test-rule',
+  alertTypeId: '.test-rule-type',
```

Comment:
Ideally it should be called ruleTypeId, but I guess `alertTypeId` is coming from a saved object, and that's why you can not change it to ruleTypeId right?

Reply:
exactly!

Reply:
It has been taken from [here](https://github.com/elastic/kibana/blob/4ada2066e3b0afc4d6a74fcb1b3c3ff7238242ab/x-pack/plugins/alerting/server/task_runner/task_runner.ts#L319)